### PR TITLE
[DNM] mgr/dashboard_v2: Add RBD mirroring API testcase

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Also:

* Changed `tox.ini` to use `mstart.sh` instaead of `vstart.sh`
* Fixed issue when rbd-mirroring is enabled without any Pools.

Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

### CODE REVIEW

File: src/pybind/mgr/dashboard_v2/tests/test_rbd_mirroring.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        rbd --cluster primary bench --io-total=32M --io-type=write --io-pattern=rand img1
+        """
+        for cmd in cmds.splitlines():
+            cmd = cmd.strip()
+            if cmd and not cmd.startswith('#'):
+                cls._cmd(cmd.split(' '))
+
+    @classmethod
+    def tearDownClass(cls):
+        cls._ceph_cmd(['osd', 'pool', 'delete', 'rbd', '--yes-i-really-really-mean-it'])
```
Comment (votdev) [inline src/pybind/mgr/dashboard_v2/tests/test_rbd_mirroring.py]: As done in test_rbd.py you should use ``cls._ceph_cmd(['osd', 'pool', 'delete', 'rbd', 'rbd', '--yes-i-really-really-mean-it'])`` here.

**Review by votdev — DISMISSED** [review]

**Review by LenzGr — DISMISSED** [review]

**Review by votdev — APPROVED** [review]

**Review by LenzGr — APPROVED** [review]
LGTM, thank you!

**Comment by sebastian-philipp** [comment]
As we need to use the theutology framework, we decided to postpone this one. 

The fix is now a seperate PR: #140

**Comment by sebastian-philipp** [comment]
We cannot merge it the way it is right now. See https://tracker.openattic.org/browse/OP-3146

**Pull request closed without merging**